staging/src/k8s.io/apimachinery/pkg/runtime/scheme.go

```diff
@@ -79,13 +79,22 @@ type Scheme struct {
 
 	// observedVersions keeps track of the order we've seen versions during type registration
 	observedVersions []schema.GroupVersion
+
+	// schemeName is the name of this scheme.  If you don't specify a name, the stack of the NewScheme caller will be used.
```

Comment:
given how much copy/pasta of packages there is, I think always capturing the call site would be useful

Reply:
> given how much copy/pasta of packages there is, I think always capturing the call site would be useful

Done.  Such a lack of trust.